packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -131,11 +133,11 @@ function Row(props) {
         rowIndex={cellRowIndex}
         columnIndex={cellColumnIndex}
         key={`${rowId}_${columnId}`}
-        isTabStop={tabStopColumnIndex === cellColumnIndex}
-        isSelected={!hasRowSelection && selectedCellColumnId === columnId}
         isMasked={cellData.isMasked}
-        isSelectable={cellData.isSelectable}
         isRowHeader={isRowHeader}
+        isSelectable={cellData.isSelectable}
```

Comment:
Why do these changes need to be made?  While we are in the midst of development, I think it would be best to limit these types of unnecessary changes to avoid merge conflicts.  It does not appear that any of these changes alter the logic.

Reply:
Reverted: [d2870fe](https://github.com/cerner/terra-framework/pull/1710/commits/d2870fedaecf3d9ca23a1a7443b5b2575389ba88)